src/ontology/mondo-edit.obo

```diff
@@ -419921,11 +419924,15 @@ xref: Orphanet:314655 {source="MONDO:equivalentTo"}
 xref: SCTID:768555009 {source="MONDO:equivalentTo"}
 xref: UMLS:C4708510 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1636705"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
+is_a: MONDO:0015159 {source="https://orcid.org/0000-0001-9310-0163"} ! multiple congenital anomalies/dysmorphic syndrome-intellectual disability
```

Comment:
Not incorrect, but I think it would be better to add the parent term as a SC of "multiple congenital (...) intellectual disability"